geometry_msgs/Pose2D pose
sensor_msgs/Image image
int8 Mobot_ID

